src/main/java/core/basesyntax/AreaCalculation.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface AreaCalculation {
```

Comment:
interface name should not sound like an action
```suggestion
public interface AreaCalculator {
```